index.js

```diff
@@ -27,13 +28,21 @@ const {
 
 const app = express();
 const port = PORT || 3000;
+const sess = {
+  secret: GRAVITY_SECRET,
+  cookie: {},
+};
 
 app.use(newrelic);
 
 if (NODE_ENV === 'production') {
   app.set('forceSSLOptions', { trustXFPHeader: true }).use(forceSSL);
+  app.set('trust proxy', 1);
```

Comment:
In order to enable secure cookies in production. It's an option that needs to be set in case the server is running behind a proxy (which it may be on heroku but I can double check). You can read about it in more details [here](https://github.com/expressjs/session/blob/master/README.md#cookiesecure)

https://github.com/expressjs/session/blob/master/README.md#cookiesecure